docs/matchers/regex.md

```diff
@@ -14,6 +16,52 @@ async def hello(opsdroid, config, message):
 
 The above skill would be called on any message which matches the regex `'hi'`, `'Hi'` or `'HI`. The `case_sensitive` kwarg is optional and defaults to `True`. 
 
+
+## Example 2
+
+```python
+from opsdroid.matchers import match_regex
+
+@match_regex('cold')
+async def is_cold(opsdroid, config, message):
+    await message.respond('it is')
+```
+
+The above skill would be called on any message that matches the regex `'cold'` . 
+
+> user: is it cold?
+>
+> opsdroid: it is
+
+Undesired effect: 
+
+> user:  Wow, yesterday was so cold at practice!
+>
+> opsdroid: it is.
+
+Since this matcher searches a message for the regular expression used on a skill, opsdroid will trigger any mention of the `'cold'`. To prevent this position anchors should be used.
+
+#### Fixed example
+```python
+from opsdroid.matchers import match_regex
+
+@match_regex('cold$')
+async def is_cold(opsdroid, config, message):
+    await message.respond('it is')
+```
+
+Now this skill will only be triggered if `'cold'` is located at the end of the message.
```

Comment:
This wouldn't actually work as the end of the message is `cold?` not `cold`. You either need to remove the question mark from the example or add the question mark to the regex. I'd probably remove the question mark.